django/chat/templates/chat/modals/advanced_qa_modal.html

```diff
@@ -1,161 +1,262 @@
 {% load i18n %}
 
-<div class="modal modal-xl fade"
-     id="advanced-qa-modal"
-     tabindex="-1"
-     aria-labelledby="advanced-qa-modal-title"
-     aria-hidden="true">
+<div
+  class="modal modal-xl fade"
```

Comment:
Formatter issues. Please ensure djLint is installed and enabled, then save the HTML files again